api/mysagw/conftest.py

```diff
@@ -25,24 +26,52 @@ def register_module(module):
 register_module(importlib.import_module(".snippets.factories", "mysagw"))
 
 
+def _get_claims(
+    settings,
+    id_claim="00000000-0000-0000-0000-000000000000",
+    groups_claim=None,
+    email_claim="test@example.com",
+):
+    groups_claim = groups_claim if groups_claim else []
+    return {
+        settings.OIDC_ID_CLAIM: id_claim,
+        settings.OIDC_GROUPS_CLAIM: groups_claim,
+        settings.OIDC_EMAIL_CLAIM: email_claim,
+    }
+
+
+@pytest.fixture
+def get_claims(settings):
+    return partial(_get_claims, settings)
+
+
+@pytest.fixture
+def claims(settings):
+    return _get_claims(settings)
+
+
 @pytest.fixture
-def admin_user(settings):
+def admin_user(settings, get_claims):
     return OIDCUser(
-        "sometoken", {"sub": "admin", settings.OIDC_GROUPS_CLAIM: ["admin"]}
+        "sometoken",
+        get_claims("admin", [settings.ADMIN_GROUP], "admin@example.com"),
```

Comment:
I suggest using named parameters here. Doing it indirectly via `partial()` causes readers to go back and forth a few times before understanding what's what

Reply:
done